diff --git a/dist/index.js b/dist/index.js
index 8cee892269bb0768d9be4f3f6407a59e3b56af40..3d9bcf418e894a9b593fbeb5f9e7e8375a1351bd 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,6 +1,6 @@
+import fs from 'fs';
 import { toMarkdown } from 'mdast-util-wiki-link';
 import { codes } from 'micromark-util-symbol/codes.js';
-import fs from 'fs';
 import path from 'path';
 
 function _iterableToArrayLimit(arr, i) {
@@ -357,11 +357,12 @@ function fromMarkdown() {
     current.data.alias = alias;
   }
   function exitWikiLink(token) {
-    var wikiLink = this.exit(token);
+    var wikiLink = top(this.stack);
     var _wikiLink$data = wikiLink.data,
       isEmbed = _wikiLink$data.isEmbed,
       target = _wikiLink$data.target,
       alias = _wikiLink$data.alias;
+    this.exit(token);
     // eslint-disable-next-line no-useless-escape
     var wikiLinkWithHeadingPattern = /^((?:[\0-\t\x0B\f\x0E-\u2027\u202A-\uD7FF\uE000-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF])*?)(#(?:[\0-\t\x0B\f\x0E-\u2027\u202A-\uD7FF\uE000-\uFFFF]|[\uD800-\uDBFF][\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])|(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF])*)?$/;
     var _target$match = target.match(wikiLinkWithHeadingPattern),
